django/contrib/contenttypes/fields.py

```diff
@@ -357,7 +357,7 @@ def __init__(
 
     def check(self, **kwargs):
         return [
-            *super().check(**kwargs),
+            *self._check_field_name(),
```

Comment:
After the fix to make the `GenericRelation` checks run, the test suite showed many errors: fields.E304, fields.E305, and fields.E311. For example:

```
aggregation_regress.HardbackBook.tags: (fields.E311) 'ItemTag.book_ptr' must be unique because it is referenced by a foreign key.
    HINT: Add unique=True to this field or add a UniqueConstraint (without condition) in the model Meta.constraints.

generic_relations_regress.Cafe.links: (fields.E304) Reverse accessor 'Link.places' for 'generic_relations_regress.Cafe.links' clashes with reverse accessor for 'generic_relations_regress.Place.links'.
    HINT: Add or change a related_name argument to the definition for 'generic_relations_regress.Cafe.links' or 'generic_relations_regress.Place.links'.
```

I think these are false positives. The easiest way to prevent them was to modify `GenericRelation.check()` to not call `super()`, like `GenericForeignKey.check()`. I tried to look through all existing checks for what’s relevant, and the field name check is the only one that really stood out.